pages/agent/v3/tokens.md

```diff
@@ -3,94 +3,25 @@
 The Buildkite Agent requires an agent token to connect to Buildkite and register for work. If you are an admin of your Buildkite organization, you can view the tokens on your [Agents page](https://buildkite.com/organizations/-/agents).
 
 
-## The default token
+## Finding and creating tokens
 
-When you create a new organization in Buildkite, a default agent token is created. This token can be used for testing and development, but it's recommended to [create new, specific tokens](#creating-tokens) for each new environment.
+When you create a new organization in Buildkite, a default agent token is created. This token can be used for testing and development, but it's recommended to create new, specific tokens for each new environment.
+
+Tokens can be found and managed on your [Agent Tokens page](https://buildkite.com/organizations/-/agent-tokens).
```

Comment:
Let's use the active voice here:

```suggestion
You can find and manage your tokens on the [Agent Tokens page](https://buildkite.com/organizations/-/agent-tokens).
```

Reply:
Is it expected that I get a 404 when I go to this URL?

Reply:
Yeah there's no central "agent tokens" page any more, it's within the [Clusters](https://buildkite.com/organizations/-/clusters) area. This docs page probably needs a little rework to point at the area and how to find tokens within.